cortex_xdr_client/api/models/action_status.py

```diff
@@ -1,11 +1,11 @@
-from typing import Dict, Optional
+from typing import Dict, Optional, Union
 
 from pydantic import BaseModel
 
 
 class ActionStatuStr(BaseModel):
     # Since we don't know what the returned key of <agent ID>/<endpoint ID> will be.
-    __root__: Dict[str, str]
+    __root__: Dict[Union[str, None], Union[str, None]]
```

Comment:
Ok so there are null fields within the dict?

Reply:
For example if the file retrieval action didn't find anything for a given endpoint, the return value for that endpoint would be null in the format of `{'endpoint_id':None, 'endpoint_id':Downloadable_link}`